berserker-cluster

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Convenience script to run Berserker on Apache Spark cluster
+# Asssume you have Apache Spark installed and configured locally
+
+E_NOT_BUILT=140
+E_NO_SPARK=141
+
+siva="$GOPATH/bin/siva"
+siva_build_command="go get -u gopkg.in/src-d/go-siva.v1/..."
+jar="target/scala-2.11/berserker_2.11-0.0.2.jar"
+build_command="./sbt package"
+
+
+hash java >/dev/null 2>&1 || { echo "Please install Java" >&2; exit 1; }
+
+if [[ ! -f "${jar}" ]]; then
+    echo "${jar} not found. Running build ${build_command}"
+    $build_command
+fi
+
+if [[ ! -f "${siva}" ]]; then
+    echo "go-siva binary not found in ${siva}. Installing it using ${siva_build_command}"
+    hash go >/dev/null 2>&1 || { echo "Please install Golang" >&2; exit 1; }
+    $siva_build_command
+fi
+
+
+
+sparkSubmit() {
+    if hash spark-submit 2>/dev/null; then
+        spark-submit "$@"
```

Comment:
Sounds good.

Please excuse my ignorance, but could you also point me to any good explanation on why that shell built-in should be used here? 

The only credible source I know on bash does not mention anything about it https://google.github.io/styleguide/shell.xml